indexing-service/src/main/java/io/druid/indexing/overlord/TaskLockbox.java

```diff
@@ -76,10 +78,21 @@
 
   @Inject
   public TaskLockbox(
-      TaskStorage taskStorage
+      TaskStorage taskStorage,
+      ServerConfig serverConfig
+  )
+  {
+    this.taskStorage = taskStorage;
+    this.lockTimeoutMillis = serverConfig.getMaxIdleTime().getMillis();
```

Comment:
serverConfig.getMaxIdleTime() corresponds to `druid.server.http.maxIdleTime` and is not deprecated.